hack/kcp-edge.sh

```diff
@@ -0,0 +1,253 @@
+  MAX_RETRIES=5
+  retries=0
+  status_code=0
+  while [ $status != "running" ]; do
+      if [ $retries -eq $MAX_RETRIES ]; then
+           status_code=1
+           break
+      fi
+
+      retries=$(( retries + 1 ))
+      sleep 3
+      status=$(process_running $1)
+  done
+  echo $status_code
+}
+
+
+#(7): Start the edge-mc controller
+echo "****************************************"
+echo "Started deploying kCP-EDGE infra ...."
+echo "****************************************"
+
+
+if kubectl get Workspace "$espw_name" &> /dev/null; then
+   echo "espw workspace already exists -- using it:"
+   kubectl ws "$espw_name"
+else 
+   if [ $verbosity == 1 ]; then
+        kubectl ws create "$espw_name" --enter
+
+        if [ $user_type == "dev" ]; then
+            kubectl apply -f  ../config/crds 
+            kubectl apply -f  ../config/exports
+            echo "Finished populate the espw with kcp edge crds and apiexports"
+
+        elif [ $user_type == "kit" ]; then
+            # Apply CRDs
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_customizers.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_edgeplacements.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_edgesyncconfigs.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_singleplacementslices.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/edge.kcp.io_syncerconfigs.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/crds/meta.kcp.io_apiresources.yaml
+
+            # Apply Exports
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiexport-edge.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiexport-meta.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-apiresources.meta.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-customizers.edge.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-edgeplacements.edge.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-edgesyncconfigs.edge.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-singleplacementslices.edge.kcp.io.yaml
+            kubectl apply -f https://raw.githubusercontent.com/kcp-dev/edge-mc/main/config/exports/apiresourceschema-syncerconfigs.edge.kcp.io.yaml
```

Comment:
I thought we agreed that both contributors and users should be using local copies of this content, for the sake of consistency with the rest of the system.

Reply:
Fixed - reading the yaml manifest from ./bin/examples